resources/asciidoctor/lib/elastic_compat_tree_processor/extension.rb

```diff
@@ -67,7 +67,13 @@ def process_subs(block)
   }.freeze
 
   def process_lang_override(block)
-    next_block = block.next_adjacent_block
+    # Check if the next block is a marker for the language
+    # We don't want block.next_adjacent_block because that'll go "too far"
+    # and it has trouble with definition lists.
+    next_block_index = block.parent.blocks.find_index block
+    return if next_block_index.nil?
```

Comment:
I believe it would be more idiomatic to write `return unless next_block_index`

Reply:
When I wrote it I felt like "This can only be nil or a string so I should be explicit" but I've come around to your way.